lib/endpoints/proofs.js

```diff
@@ -51,45 +51,45 @@ function formatAsChainpointV3Ops(proof, op) {
 }
 
 /**
- * GET /proofs/:hash_id handler
+ * GET /proofs/:proof_id handler
```

Comment:
Found some left over changes that needed to be made for v4 updates. Mainly just comments and error messages that were confusing in lieu of the change